src/main/java/org/zalando/planb/provider/OIDCController.java

```diff
@@ -77,6 +77,11 @@ static String getRealmName(RealmConfig realms, Optional<String> realmNameParam,
                 .orElseThrow(() -> new RealmNotFoundException(hostHeader.get())));
     }
 
+    static String getRealmName(RealmConfig realms, String realmNameParam) {
```

Comment:
I thought this made sense since to keep consistency since there's already a getRealmName method in there, but I could move it to RealmConfig